cmd/experimental/kjobctl/pkg/builder/slurm_builder.go

```diff
@@ -75,6 +75,8 @@ const (
 
   echo "$replaced"
 }`
+
+	WorkloadPriorityClassLabel = "kueue.x-k8s.io/priority-class"
```

Comment:
I think we can use this constant:

https://github.com/kubernetes-sigs/kueue/blob/12fad5f72ba85f98d83b6557252f6b68de711e5b/pkg/controller/constants/constants.go#L35-L38